meteor/lib/collections/CoreSystem.ts

```diff
@@ -89,6 +89,13 @@ export interface ICoreSystem {
 		transactionSampleRate?: number
 	}
 	enableMonitorBlockedThread?: boolean
+
+	/** Cron jobs running nightly */
+	cron?: {
+		casparCG?: {
```

Comment:
it would be clearer to call this `casparCGRestart`

Reply:
I renamed it to `casparCGRestart`